lib/parser.js

```diff
@@ -63,7 +63,7 @@ exports.filter = function filter(options, done) {
     let q = query.q;
 
     //check if is search query
-    const filters = _.omitBy(_.merge({}, filter, { q }), val => !val);
+    const filters = _.omitBy(_.merge({}, filter, { q }), val => val === undefined);
```

Comment:
@codesxt Please also check for `null`, `''`, `' '`, `NaN` i.e falsey value excluding booleans

Reply:
Done. Please check.